client/pages/project-details/details.template.html.slim

```diff
@@ -25,6 +30,9 @@
               dd {{ project.createdAt | amDateFormat:'llll' }}
               dt Last Update
               dd {{ project.updatedAt | amDateFormat:'llll' }}
+              dt ng-if='project.repoUrl' Code repository
+              dd ng-if='project.repoUrl'
+                a href='{{ project.repoUrl }}' target='_blank' {{ project.repoUrl }}
```

Comment:
This doesn't look good when the URL is too long, since URLs don't wrap. We should add some CSS to truncate if it overflows.